pages/linux/apt-file.md

```diff
@@ -0,0 +1,15 @@
+# apt-file
+
+> Search for files in apt packages, including ones not yet installed.
+
+- Update the metadata database:
+
+`sudo apt update`
+
+- Search for packages that contain the specified file:
+
+`apt-file search {{part/of/filename/or/path}}`
```

Comment:
In the description, you mention "file", but in the example, you add that it can be a part of the filename too. I am wondering if this info can be moved to the description, and the example be kept simple.

Reply:
I see. Doing some research reveals that it uses a substring search. How about this?

```
Search for packages that contain the specified file or path
```

Reply:
SGTM

Reply:
Did you want to change the token too, or keep it the same ?

Reply:
Oops! Forgot about it :P